src/components/ContactsList/ContactRow.jsx

```diff
@@ -166,10 +96,12 @@ ContactRow.propTypes = {
   selection: PropTypes.shape({
     selected: PropTypes.bool,
     onSelect: PropTypes.func
-  })
+  }),
+  onClick: PropTypes.func
 };
 ContactRow.defaultProps = {
-  selection: null
+  selection: null,
+  onClick: null
```

Comment:
Cette prop ne donne aucune information de ce qu'on peut faire. `onClick` est un nom d'événement, pas un nom de propriété ou de fonctionnalité. Par exemple dans [thinking in react](https://reactjs.org/docs/thinking-in-react.html), on a une prop `onFilterTextChange` : 

```jsx
<SearchBar
    filterText={this.state.filterText}
    inStockOnly={this.state.inStockOnly}
    onFilterTextChange={this.handleFilterTextChange}
    onInStockChange={this.handleInStockChange}
/>
```

Reply:
Je suis d'accord sur le principe, mais le composant `ContactRow` n'a aucune idée de ce qu'il se passe quand on lui clique dessus, et de fait il pourrait se passer des choses différentes selon les situations. Donc je ne vois pas de nom plus pertinent que `onClick`.

Reply:
Je pense que `<ContactRow onClick={} />` est mieux que `<ContactRow onRowClick={} />`. Je verrais un interêt à préciser sur quoi on écoute un clic si ce n'était pas sur l'élément parent (par exemple `onClickEmail`), mais si c'est sur le parent `onClick` me semble le plus clair.

Reply:
or `onClickContact` like https://github.com/cozy/cozy-contacts/pull/32/files#diff-b8c853e56cc8b58d63005d12ea0c27c1R70